sklearn/linear_model/tests/test_logistic.py

```diff
@@ -685,11 +699,13 @@ def test_ovr_multinomial_iris():
         assert scores.shape == (3, n_cv, 10)
 
 
-def test_logistic_regression_solvers():
+def test_logistic_regression_solvers(global_random_seed):
     """Test solvers converge to the same result."""
-    X, y = make_classification(n_features=10, n_informative=5, random_state=0)
+    X, y = make_classification(
+        n_samples=200, n_features=10, n_informative=5, random_state=global_random_seed
+    )
 
-    params = dict(fit_intercept=False, random_state=42)
+    params = dict(C=0.1, fit_intercept=False, random_state=global_random_seed)
```

Comment:
Is it better to skip setting the `global_random_seed` for the solvers than don't use it, but make the code a bit more complex? or to set it to all of them but make the test take longer?